sync-core/src/test/java/com/cloudant/sync/query/IndexManagerTest.java

```diff
@@ -30,6 +36,67 @@
 public class IndexManagerTest extends AbstractIndexTestBase {
 
     @Test
+    public void correctlyMigratesToNewMetadataSchema() {
```

Comment:
With the updated incremental manner in which the metadata table is being created (and altered) there is no longer a need for this test.  Removed in c716a18a71a0afe6dd681cfd3843746b60847d80.